### ISSUE TITLE
Burger's Equation

### ISSUE THREAD

**Issue by candyartemis** [maziarraissi/PINNs#15]
Hello,

I was trying to run the continuous Burger's equation file, but I do not get the figure in the folder, instead, I keep getting the figure that the trained solution is a horizontal red dash line. which is nowhere close to the true solution. I checked the training step prints, and the loss value does not change with time. Does anyone else get the same thing?

Thanks.

**Comment by candyartemis** [comment]
appendix/continuous_time_inference (Burgers)/Burgers.py
I got the same figure with the identification one too

**Comment by Ankitkumar803** [comment]
same here
